hvplot/plotting/scatter_matrix.py

```diff
@@ -4,10 +4,20 @@
 
 from ..converter import HoloViewsConverter
 from ..util import with_hv_extension
+from holoviews.operation.datashader import datashade as _datashade 
+from holoviews.operation.datashader import spread as _spread 
+from holoviews.operation.datashader import dynspread as _dynspread 
+from holoviews.operation.datashader import rasterize as _rasterize 
```

Comment:
I think it would be slightly cleaner to do:

```suggestion
import holoviews.operation.datashader as hd 
```

And then you could use `hd.rasterize`, `hd.spread`, etc. below. That way I wouldn't have to wonder if the meaning of _spread was changed somewhere between here and where it's used, as it's just standard namespace usage.